frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx

```diff
@@ -82,6 +97,20 @@ const DashboardInfoSidebar = ({
         />
       </ContentSection>
 
+      <ContentSection>
+        <FormField
+          title={t`Auto-apply filters`}
+          titleStyle="large"
+          orientation="horizontal"
+          noMargin
+        >
+          <Toggle
+            aria-label={t`Auto-apply filters`}
```

Comment:
I couldn't find a nice way to use `FormField`'s `title` for its children's `aria-label`. I notice it accepts `htmlFor`, but that includes the error message as well. 
https://github.com/metabase/metabase/blob/6476e38ac0b33fcc3a1ac069fd97b01347414441/frontend/src/metabase/core/components/FormField/FormField.tsx#L73-L80
 I don't want to be touching that part yet, but I believe the input label should be static and not depend on the state of the input. i.e. it should not change when the input has an error.

Reply:
If I plainly remove `aria-label` from the `<Toggle />` [the test would simply fail](https://github.com/metabase/metabase/blob/734c39090455f64826e824ab59e51a76b999f649/e2e/test/scenarios/dashboard-filters/old-parameters.cy.spec.js#L88).
<img width="353" alt="image" src="https://user-images.githubusercontent.com/1937582/232475523-9734f8eb-6d3b-4907-a5ca-7a4f79e9077e.png">

I need to either
1. Add `aria-label` to the `<Toggle />` or
2. Set `id` to `<Toggle />` and pass that id to `<FormField />`'s `htmlFor`
    ![image](https://user-images.githubusercontent.com/1937582/232476070-175b7303-4715-4b2b-83ae-f1d5274d2bd7.png)


I think using `htmlFor` is preferable since we do not need to hard-code `aria-label` and we use an actual label from the DOM instead. What do you think?